src/pages/scoreboard.js

```diff
@@ -0,0 +1,57 @@
+import React, { useCallback, useState } from 'react';
+import { Footer } from '../components/common/footer';
+import { Header } from '../components/common/header';
+import { supabase } from '../utils/supabase';
+
+import { ScoreBoardTable } from '../components/pages/scoreboard/scoreboardTable';
+
+export const ScoreboardPage = ({ isSignedIn }) => {
+  const ref = React.useRef();
+  const [communities, setCommunities] = useState({ data: [], loading: true });
+
+  const fetchCommunities = useCallback(async () => {
+    setCommunities((d) => ({ ...d, loading: true }));
+    const { data } = await supabase.select('scoreboard');
+    const communityCounter = {};
+    data.map((item) => {
+      if (!communityCounter[item?.['community-name']]) {
```

Comment:
we should store `community-name` and other related keys in constants, since it is used in many place

Reply:
storing data constants to access data isn't really relevant!

Reply:
i m not saying to store data in constants
I m saying to store the keys name in constants
```
const CommunityDataKeys = {
COMMUNITY_NAME:"community-name",
... others
}
```
so that in future if we change it ever we don't have to update it everywhere, or if any other developer works on it, it's easier to work with the data

Reply:
oh alright got it , that we can do